website/views/shared/modals/members.jade

```diff
@@ -46,9 +46,12 @@ script(type='text/ng-template', id='modals/private-message.html')
   .modal-header
     h4=env.t('pmHeading', {name: "{{profile.profile.name}}"})
   .modal-body
-    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',ng-model='_message')
+    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',placeholder=env.t('needsTextPlaceholder'),ng-model='_message')
```

Comment:
Since we're dependent on an AJAX call to do the commit and close the form, I just stuck a quick truthy check followed by a return statement. If that's not robust enough, we can do the full null/undefined/empty string check.

Side note: I hit Ctrl-Enter on Chrome and nothing happened, even when the text box had text. I confirmed the event wasn't even firing (and was working when the button was clicked), so it wasn't the new code blocking it. Maybe a ticket that needs to be opened elsewhere? :shrug: